TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Proxy.java

```diff
@@ -46,6 +52,9 @@
     private Set<Server> registeredServers;
     private LogStorage logStorage;
     private PublicKey publicKey;
+    private int pid;
+    private final ScheduledExecutorService scheduler;
```

Comment:
Also shutdown the scheduler when the proxy is shut down

Reply:
isn't this already in line 109?
onShutdown